docs/DeveloperGuide.md

```diff
@@ -316,21 +332,20 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 (For all use cases below, the **System** is the `RC4HDB` and the **Actor** is the `user`, unless specified otherwise)
 
 
-**Use case: Getting help with the application**
+**Use case: UC1. Getting help with the application**
 
 **MSS**
-1. User needs help in utilizing RC4HDB.
-2. User uses the help command.
-3. RC4HDB displays a message that directs the user to our user guide.
+1. User needs help related to RC4HDB.
+2. RC4HDB displays a message that directs the user to our user guide.
 
     Use case ends.
 
-**Use case: Adding a single resident**
+**Use case: UC2. Add a single resident**
 
 **MSS**
-1. New resident moves into RC4.
+1. New resident moves into [**RC4**](#glossary).
 2. User gets personal details of the resident.
-3. User uses the add command to add the resident.
+3. User adds the resident.
 4. RC4HDB adds the resident to the data file.
 5. RC4HDB displays the name and other information of the resident.
 
```

Comment:
Consider adding more space between use cases with `<br.>`, but of course, without the '.'